packages/react-components/react-dialog/library/src/components/DialogSurfaceMotion.ts

```diff
@@ -1,10 +1,11 @@
 import { createPresenceComponent, motionTokens } from '@fluentui/react-motion';
-import { tokens } from '@fluentui/react-theme';
 
 const keyframes = [
-  { opacity: 0, boxShadow: '0px 0px 0px 0px rgba(0, 0, 0, 0.1)', transform: 'scale(0.85) translateZ(0)' },
```

Comment:
can we add an existing rule or create one to mitigate this in future ? there are known css props that are performance nightmare if someone tries to use them for any kind of animations.